view/frontend/web/js/model/adyen-payment-service.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright © 2015 Magento. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+define(
+    [
+        'jquery',
+        'underscore',
+        'Magento_Checkout/js/model/quote',
+        'Magento_Checkout/js/model/url-builder',
+        'Adyen_ExpressCheckout/js/helpers/getIsLoggedIn',
+        'mage/storage'
+    ],
+    function(
+        $,
+        _,
+        quote,
+        urlBuilder,
+        getIsLoggedIn,
+        storage
+    ){
+        'use strict';
+
+        function paymentDetails(data, orderId, isMultishipping = false, quoteId = null) {
```

Comment:
Multishipping is not a valid use case for express module. Therefore, this argument is not required. Could you please update the arguments?